### ISSUE TITLE
Allow/document steps for customizing the notification

### ISSUE THREAD

**Issue by ColtonIdle** [ChuckerTeam/chucker#406]
## :warning: Is your feature request related to a problem? Please describe
I'd like to suppress the notification icon from appearing in the status bar, but still have it be apart of the notification panel. Also, right now clicking on the notification removes it from the panel. It would be nice to have an option to have it stay there.

**Comment by vbuberen** [comment]
I remember another question and request about notifications in #154 
I think we should implement more modes than just show/hide. 

For me it seems that we need 2 modes available for configuration:
1. Having a persistent non-dismissible notification in the notification shade.
2. Current style with dismissible notifications.

Both have their right for existence, since for some users, like me, notification is often an indicator that some network activity started which I often rely on.

**Comment by ColtonIdle** [comment]
Yeah, I think everyone has their own idea about the notification from chucker. For me, I want a notification in the tray that doesn't go away when you select it, but it goes away on swipe, and it does not show in the status bar.

**Comment by cortinico** [comment]
Definitely makes sense to consider. We should probably collect all the requested notification "modes".
So far the `ChuckerCollector` has a parameter `showNotification` that will toggle the notification visibility. Ideally we could replace that with a sealed class hierarchy (or an enum) where the user can select the desired mode. (This is obviously going to end up in 4.x).
